ext/opcache/shared_alloc_mmap.c

```diff
@@ -40,6 +40,340 @@
 # define MAP_HUGETLB MAP_ALIGNED_SUPER
 #endif
 
+#if defined(__x86_64__) && defined(__linux__)
+#define max(a,b) ((a) > (b) ? (a):(b))
+
+/* This function must exist in PHP .text segment */
+void* php_text_lighthouse(void);
```

Comment:
Any reason for re-declaring it here ? (it's in zend.h as well)

Reply:
I will check if zend.h is  included implicitly or explicitly. Update here soon

Reply:
Thanks for pointing this out. I did not realize zend.h has already included by "zend_shared_alloc.h" file. will remove this duplicated declaration.